recipes/stringtie/1.0.3/meta.yaml

```diff
@@ -4,7 +4,7 @@ about:
   summary: Transcriptome assembly and quantification for RNA-seq
 build:
   number: 1
-  skip: True # [osx]
+  skip: True  # See build issues in https://github.com/bioconda/bioconda-recipes/pull/9460/
```

Comment:
can you put those recipes into your blacklist?

Reply:
Where do you want me to list them, specifically?

Reply:
build-fail-blacklist?